packages/database/src/migrations/20210722044503-FixDataSourceEntityTypeLesmisProvinceStudentDashboards-modifies-data.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboards = [
+  'LESMIS_gross_intake_ratio_primary_province',
+  'LESMIS_gross_intake_ratio_lower_secondary_province',
+  'LESMIS_gross_intake_ratio_upper_secondary_province',
+];
+
+exports.up = function (db) {
+  return db.runSql(`
+    UPDATE report
+    SET config = jsonb_set(config, '{fetch,aggregations,0,config,dataSourceEntityType}','"district"', false)
```

Comment:
Is this gonna have the same issue on the country level reports?

Reply:
Yep I believe you're right, good catch! @rohan-bes I'll update the migration now